### PR TITLE
chore: use the crunchydata postgres operator

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: app/server/db/index.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const getDatabaseUrl = () => {
 
   databaseURL += PGUSER;
   if (process.env.PGPASSWORD) {
-    databaseURL += `:${process.env.PGPASSWORD}`;
+    databaseURL += `:${encodeURIComponent(process.env.PGPASSWORD)}`;
```
Comment (matthieu-foucault) [inline app/server/db/index.ts:17]: the passwords generated by the postgres operator may have special characters in them

File: chart/cas-cif/values.yaml [hunk 2]
```diff
@@ -49,7 +30,7 @@ app:
 
 db:
   name: cif
-  appUser: cif_app
+  appUser: cifapp
```
Comment (matthieu-foucault) [inline chart/cas-cif/values.yaml:33]: crunchydb doesn't like underscores in its user names

File: chart/cas-cif/templates/cronJobs/db-init.yaml [hunk 3]
```diff
@@ -14,62 +12,35 @@ spec:
...
               image: {{ .Values.image.psql.repository }}:{{ default .Values.defaultImageTag .Values.image.psql.tag }}
               imagePullPolicy: {{ default .Values.defaultImagePullPolicy .Values.image.psql.pullPolicy }}
               resources:
                 limits:
-                  cpu: 1000m
+                  cpu: 50m
                   memory: 128Mi
                 requests:
-                  cpu: 100m
+                  cpu: 10m
```
Comment (pbastia) [inline chart/cas-cif/templates/cronJobs/db-init.yaml:35]: any risk in getting this so low?

**Review by pbastia — APPROVED** [review]
looks amazing 👌

File: chart/cas-cif/templates/cronJobs/db-init.yaml (hunk 3 above)
Comment (matthieu-foucault) [reply chart/cas-cif/templates/cronJobs/db-init.yaml:35]: nah this just running psql and doesn't need any CPU

File: app/server/db/index.ts (hunk 1 above)
Comment (pbastia) [reply app/server/db/index.ts:17]: weren't we able to pull that URL from the secret generated by the operator? that might be a tech debt item to keep it compatible with local dev

**Review by pbastia — APPROVED** [review]

**Pull request merged**